content/applications/hr/recruitment/new_job.rst

```diff
@@ -143,68 +149,241 @@ Sections are indicated by a gray background, while questions have a white backgr
    :align: center
    :alt: A sample of categories and questions for a candidate.
 
-Next, configure the options for the interview form. Click the :guilabel:`Options` tab to view all
-the options to configure, by category.
+Answers
+*******
+
+If :guilabel:`Multiple choice: only one answer`, :guilabel:`Multiple choice: multiple answers
+allowed`, :guilabel:`Single Line Text Box`, or :guilabel:`Matrix` is selected for the
+:guilabel:`Question Type`, an :guilabel:`Answers` tab appears beneath the question. If another
+:guilabel:`Question Type` is selected, the :guilabel:`Answers` tab remains hidden from view.
+
+Multiple choice
+^^^^^^^^^^^^^^^
+
+For both the :guilabel:`Multiple choice: only one answer` and :guilabel:`Multiple choice: multiple
+answers allowed` :guilabel:`Question Types`, the answers are populated in the same way.
+
+First, in the :guilabel:`Answers` tab, click :guilabel:`Add a line`. A line appears, and an answer
+can be entered. After typing in the answer, click off the line, or press enter to lock in the new
+answer on the form and have another answer line appear.
+
+If desired, an image can be attached to the answer. Click on a line to select it, and an
+:guilabel:`Upload your file` button appears on the right side. Click the :guilabel:`Upload your
+file` button and a file explorer window appears. Navigate to the photo file, select it, then click
+:guilabel:`Open` to select it.
+
+Repeat this for all the answers to be added for the multiple choice question. The answers can be
+rearranged in any order. To move an answer, click on the :guilabel:`↕️ (Up-down arrows)` on the far
+left of each answer line, and drag the answer to the desired position. The order the answers appear
+in the form is the order the answers will appear online.
+
+To delete a line, click on the :guilabel:`🗑️ (trash can)` icon on the far right side of the answer
+line.
+
+.. image:: new_job/multi-answers.png
+   :align: center
+   :alt: Answers to a multiple choice question, where each line has a different answer listed.
+
+Single Line Text Box
+^^^^^^^^^^^^^^^^^^^^
+
+If the :guilabel:`Single Line Text Box` is selected for the :guilabel:`Question Type`, only two
+checkboxes appear in the :guilabel:`Answers` tab:
+
+- :guilabel:`Input must be an email`: Activate this option if the answer must be in the format of an
+  email address.
+
+  - :guilabel:`Save as user email?`: This option appears if :guilabel:`Input must be an email` is
+    selected. This saves the email entered on the form as the user's email, and will be used anytime
+    Odoo contacts the user via email.
+
+- :guilabel:`Save as user nickname?`: Activate this option to populate the answer as the user's
+  nickname. This is stored and used anywhere Odoo uses a nickname.
```

Comment:
It is harder to tell from my perspective, but the formatting on the bulleted lists just looks a bit off. When you ran make, did it format correctly? (If so, you can ignore this, but I wanted to call it out just in case. The spacing just looks like is it weird)

Reply:
Yes- but that's because the second bullet is nested in (indented) from the one above because it only appears if it is checked off.  So it looks off here, but that's why- they are not all 3 lined up together.